CP-CPS.md

```diff
@@ -190,23 +190,19 @@ Read only access to the Policy and Legal Repository and certificate information
 
 ### 3.1.1 Types of names
 
-Certificate distinguished names and subject alternative names are compliant with the Baseline Requirements.
+See Section 7.1 for the kinds of names which may appear in Let's Encrypt certificates.
```

Comment:
In the title of this section it says "types" but in this edit we're saying "kinds" - is it better to use the title word, "types" in the content?

Reply:
Good point, done.